schema/deploy/shipping/views.sql

```diff
@@ -138,27 +138,27 @@ create or replace view shipping.return_results_v1 as
              when sample_id is not null and count(present) = 0 then 'processing'
              when count(present) > 0 then 'complete'
            end as status,
-           -- We only return the top level organisms for results so we want to omit subtypes
-           array_agg(distinct subpath(organism, 0, 1)::text)
-            filter (where present and organism is not null)  as organisms_present,
-           array_agg(distinct subpath(organism, 0, 1)::text)
-            filter (where not present and organism is not null) as organisms_absent
+           array_agg(distinct organism::text) filter (where present and organism is not null)  as organisms_present,
+           array_agg(distinct organism::text) filter (where not present and organism is not null)  as organisms_absent
 
       from warehouse.identifier
       join warehouse.identifier_set using (identifier_set_id)
       left join warehouse.sample on uuid::text = sample.collection_identifier
-      left join shipping.presence_absence_result_v1 on sample.identifier = presence_absence_result_v1.sample
+      left join shipping.presence_absence_result_v2 on sample.identifier = presence_absence_result_v2.sample
 
-    --These are all past and current collection identifier sets not including self-test
+    --These are all past and current collection identifier sets not including Y1 Flu@Home self-test
     where identifier_set.name in ('collections-seattleflu.org',
                                   'collections-kiosks',
                                   'collections-environmental',
                                   'collections-swab&send',
+                                  'collections-self-test',
                                   'collections-household-observation',
                                   'collections-household-intervention') and
           (organism is null or
           -- We only return results for these organisms, so omit all other presence/absence results
-          organism <@ '{"Adenovirus", "Human_coronavirus", "Enterovirus", "Influenza", "Human_metapneumovirus", "Human_parainfluenza", "Rhinovirus", "RSV"}'::ltree[])
+          organism ? '{"Influenza.A", "Influenza.B", "RSV"}'::lquery[]) and
+          -- Limit to only Cepheid results
+          (presence_absence_result_v2.details is null or presence_absence_result_v2.details @> '{"device":"Cepheid"}')
```

Comment:
It may not matter in practice, but there might be non-Cepheid results with null details and they would be included by this condition. To detect the case of no matching row for the left join, better to condition on the PK or equivalent of the joined table, e.g. `presence_absence_result_v2.sample is null`.